operator/v1/types_console.go

```diff
@@ -202,6 +205,49 @@ type AddPage struct {
 	DisabledActions []string `json:"disabledActions,omitempty"`
 }
 
+type PerspectiveState string
+
+// These are valid perspective states. "Enabled" means the perspective is shown.
+// "Disabled" means the Perspective is hidden.
+// "AccessReview" means access review check is required to show or hide a Perspective.
+const (
+	Enabled      PerspectiveState = "Enabled"
+	Disabled     PerspectiveState = "Disabled"
+	AccessReview PerspectiveState = "AccessReview"
+)
+
+type PerspectiveAccessReview struct {
+	// requiresAccessReview defines a list of permission checks. The perspective will only be shown when all checks are successful.
```

Comment:
What happens when there are no checks?

What happens when this is configured and `missingAccessReview` is also configured?

Since this is a list, should it be plural rather than singular?

Reply:
If there are no checks, we show the perspective. Added details as per the enhancement doc

updated it to `required` and `missing`